cli.js

```diff
@@ -0,0 +1,36 @@
+let colors = require("colors");
+
+const { readFile, readFileValidation, readFilesStats } = require("./index.js");
```

Comment:
El uso del let es recomendado para el momento en que una variable muta
y const para cuando no muta el valor

```suggestion
cost colors = require("colors");

const { readFile, readFileValidation, readFilesStats } = require("./index.js");
```